README.md

```diff
@@ -1,10 +1,15 @@
+# Neighbourhood Collaboration Site
+
+## A system to keep track of people, houses, and addresses of those houses.
+
+
 1. Consider the type of data we will be storing and therefore the type of database we should implement (SQL vs NoSQL)
 
 - Relational database with SQL
 
 2. Create a schema for this database
 
-
+![alt text](schema.png)
```

Comment:
I didn't know how to add images to markdown thanks! Very cool

A couple of things we wanted to say about the schema:
- Neither table contains a common key through which to join the tables
- Owner/name seems to conflict in house table and people table?